index.html

```diff
@@ -277,27 +277,30 @@ <h3 class="my_name">Get Started</h2>
               a feature that you need built or a project that needs coding. 
               I'd love to help with it. </p>
 
-            <form class="form_collab" action="https://formspree.io/f/xaykvgeq" method="POST">
+            <form class="form_collab" action="https://formspree.io/f/xknepwnz" method="POST" id="form_all">
               <ul class="list_forms">
                 <div class="name_email">
                   <li class="form_input">
                     <label>
                       Your name<input placeholder="Your Name" type="text" id="name"
-                        name="name" maxlength="30" required>
+                        name="name" maxlength="30" required pattern="([A-ZΆ-ΫÀ-ÖØ-Þ][A-ZΆ-ΫÀ-ÖØ-Þa-zά-ώß-öø-ÿ]{1,19} ?){1,10}" onchange="storeForm()">
                     </label>
                   </li>
+                  <!-- <p id="reply_name"></p>  -->
                   <li class="form_input">
                     <label>
                       Your email
                       <input
-                        placeholder="Your Email"
-                        id="mail" type="email" name="email" required>
+                        placeholder="Your Email" class="email_bg"
+                        id="mail" type="email" placeholder="email@domain.com" name="email" required>
                     </label>
                   </li>
+                   <!-- <p id="reply_mail"></p>  -->
```

Comment:
- [ ] KIndly make sure you remove all the commented and unused code lines from your code base for **[best practice reasons](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)**. This makes your project look messy.